app/components/preview-line/template.hbs

```diff
@@ -0,0 +1,6 @@
+{{yield}}
+{{#if editMode}}
+	<div class="line" style="border-bottom: 2px dashed {{layer.settings.color}};">
+		<span class="line-text" style="background-color: {{layer.settings.bgColor}};">{{text}}</span>
+	</div>
+{{/if}}
```

Comment:
\n

Reply:
Also we should keep moving the styles out.

Reply:
WIll do in later PR